src/callback_handler/kubernetes/client.rs

```diff
@@ -295,15 +297,37 @@ impl Client {
 
     pub async fn can_i(
         &mut self,
-        subject_access_review: &SubjectAccessReview,
+        user: String,
+        group: String,
+        namespace: String,
+        resource: String,
+        verb: String,
     ) -> Result<SubjectAccessReviewStatus> {
+        let subject_access_review = SubjectAccessReview {
+            spec: SubjectAccessReviewSpec {
```

Comment:
I would go into the SDK and implement the `From<SubjectAccessReview> for SubjectAccessReviewSpec` trait. This would allow us to do something like:

```rust
SubjectAccessReview {
  spec: access_review.into(),
}
```

BTW, I just realized that we named our type `SubjectAccessReview`, which would conflict/cause confusion with the same type defined by the `k8s-openapi` crate.
This is fine, I think we should leave the nice name inside of the SDK. We can just rename our own type here as soon as we import it:

```rust
use kW_sdk::SubjectAccessReview as KWSubjectAccessReview;
```